src/desktop/apps/auction/__tests__/utils/renderTestComponent.js

```diff
@@ -1,4 +1,11 @@
-import 'jsdom-global/register'
+// Include JSDom if running from Mocha. Jest automatically injects.
+if (
+  typeof navigator === 'undefined' ||
+  !navigator.userAgent.includes('jsdom')
+) {
```

Comment:
@erikdstock - can you try this instead? This is a little more clear to outside devs: 

```javascript
const isMocha = typeof jest === "undefined"`

if (isMocha) {
  require('jsdom-global/register')
}
```